src/main/java/be/syntra/devshop/DevshopFront/services/utils/CategoryMapperUtil.java

```diff
@@ -14,4 +15,10 @@
         categories.forEach(category -> categoryNames.add(category.getName()));
         return categoryNames;
     }
+
+    public List<Category> mapToCategories(List<String> categoryStrings) {
+        List<Category> categories = categoryStrings.stream().map(categoryString -> Category.builder().name(categoryString).build()).collect(Collectors.toList());
```

Comment:
stream over multiple lines (see previous comment).
Also, you can inline this stream. Simply: return the stream, no need to create a variable

Reply:
split + inlined